packages/web/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Do we need this?

Reply:
Yep, it's part of vite test app, but it can be moved to a subfolder I guess.

Reply:
Looks like moving to subfolder is not so easy, so I'll leave it here